js/mainjs/config.js

```diff
@@ -1,12 +1,13 @@
 'use strict';
 
-// The default settings 
+// The default settings
 const defaultConfig = {
 	homePlugin:  'Overview',
 	siad: {
     	path: require('path').join(__dirname, '../..', 'Sia'),
 		detached: false,
 	},
+	persistInTray: true,
```

Comment:
can we call this `closeToTray` instead?
Also, I'm thinking that since linux users may not have a tray, this should be `false` by default for linux users:
```js
// Linux users may not have a system tray
closeToTray: process.platform !== 'linux',
```